app/components/primer/beta/blankslate.rb

```diff
@@ -0,0 +1,241 @@
+      #       <% c.description { "Wikis provide a place in your repository to lay out the roadmap of your project, show the current status, and document software better, together."} %>
+      #       <% c.primary_action(href: "https://github.com/monalisa/mona/wiki/_new").with_content("Create the first page") %>
+      #       <% c.secondary_action(href: "https://docs.github.com/en/github/building-a-strong-community/about-wikis").with_content("Learn more about wikis") %>
+      #     <% end %>
+      #
+      # @example Variations
+      #   @description
+      #     There are a few variations of how the Blankslate appears: `narrow` adds a maximum width, `large` increases the font size, and `spacious` adds extra padding.
+      #   @code
+      #     <%= render Primer::Beta::Blankslate.new(
+      #       narrow: true,
+      #       large: true,
+      #       spacious: true,
+      #     ) do |c| %>
+      #       <% c.graphic_icon(icon: :book) %>
+      #       <% c.title(tag: :h2).with_content("Welcome to the mona wiki!") %>
+      #       <% c.description { "Wikis provide a place in your repository to lay out the roadmap of your project, show the current status, and document software better, together."} %>
+      #     <% end %>
+      #
+      # @example With border
+      #   @description
+      #     It's possible to add a border around the Blankslate. This will wrap the Blankslate in a BorderBox.
+      #   @code
+      #     <%= render Primer::Beta::Blankslate.new(border: true) do |c| %>
+      #       <% c.graphic_icon(icon: :book) %>
+      #       <% c.title(tag: :h2).with_content("Welcome to the mona wiki!") %>
+      #       <% c.description { "Wikis provide a place in your repository to lay out the roadmap of your project, show the current status, and document software better, together."} %>
+      #     <% end %>
+      #
+      # @param narrow [Boolean] Adds a maximum width to the Blankslate.
+      # @param large [Boolean] Increases the font size in the Blankslate.
+      # @param spacious [Boolean] Increases the vertical padding.
+      # @param border [Boolean] Adds a border around the Blankslate.
+      # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+      def initialize(narrow: false, large: false, spacious: false, border: false, **system_arguments)
+        @border = border
+        @system_arguments = system_arguments
+        @system_arguments[:tag] = :div
+        @system_arguments[:classes] = class_names(
+          @system_arguments[:classes],
+          "blankslate",
+          "blankslate-narrow": narrow,
+          "blankslate-large": large,
+          "blankslate-spacious": spacious
+        )
+      end
+
+      def render?
+        title.present?
+      end
+
+      def wrapper
+        unless @border
+          yield
+          return # returning `yield` caused a double render
+        end
+
+        content_tag(:div, class: "Box") do
```

Comment:
Wouldn't we want to use BorderBox here?

Reply:
`BorderBox` doesn't allow us to put stuff directly into `<div class="Box">` :/